package.json

```diff
@@ -111,7 +111,7 @@
     "silent-error": "^1.0.0",
     "symlink-or-copy": "^1.0.1",
     "temp": "0.8.3",
-    "testem": "0.9.11",
+    "testem": "testem/testem",
```

Comment:
Yes, I'm working on testem v1. The git ref is just here until the release is cut.